scripts/shared/lib/utils

```diff
@@ -105,12 +101,33 @@ function declare_cidrs() {
     # shellcheck disable=SC2034 # these variables are used elsewhere
     declare -gA cluster_CIDRs service_CIDRs global_CIDRs
 
-    for i in {1..3}; do
-        add_cluster_cidrs $i
+    i=1
+    for cluster in $(eval echo "$1"); do
+        add_cluster_cidrs "$i" "$cluster"
+        i=$(("$i"+1))
     done
 }
 
 function declare_kubeconfig() {
+    local clusters
```

Comment:
We already depend on `git` and specifically `git rev-parse` in other places.

Besides we're dogfooding our suggestion to the user this way (perhaps we can even extract this to a constant and use it in both places).

Also you could just put `${DAPPER_SOURCE}` if you wish to avoid `git rev-parse`

Reply:
yeah I can use `${DAPPER_SOURCE}`  here.